### PR TITLE
Initial port cpython to Windows 10 running on ARM32 processors

### PR DESCRIPTION
**Pull request by paulmon**

### CODE REVIEW

File: Tools/nuget/pythonARM32.nuspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>pythonx86</id>
```
Comment (jslobodzian) [inline Tools/nuget/pythonARM32.nuspec]: Is "pythonx86" correct here?  not arm?

File: Modules/_ctypes/callproc.c [hunk 2]
```diff
@@ -800,7 +803,11 @@ static int _call_function_pointer(int flags,
 #ifndef DONT_USE_SEH
     __try {
 #endif
+#ifdef _M_ARM
+        delta = 0;
+#else
         delta =
+#endif // _M_ARM
```
[the remark this answers is not in the document]
Comment (paulmon) [reply Modules/_ctypes/callproc.c]: This suppresses a compiler warning

File: Include/pyatomic.h [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 #if defined(_MSC_VER)
 #include <intrin.h>
-#include <immintrin.h>
+//#include <immintrin.h>
```
[the remark this answers is not in the document]
Comment (paulmon) [reply Include/pyatomic.h]: Not for ARM, but I'm not sure about everything else.

File: Lib/ctypes/test/test_win32.py [hunk 4]
```diff
@@ -11,6 +11,7 @@
 @unittest.skipUnless(sizeof(c_void_p) == sizeof(c_int),
                      "sizeof c_void_p and c_int differ")
 class WindowsTestCase(unittest.TestCase):
+    @unittest.skipIf(platform.machine() == 'ARM', "API not present on Windows 10 IoT Core")
```
[the remark this answers is not in the document]
Comment (paulmon) [reply Lib/ctypes/test/test_win32.py]: IsWindows is supported on desktop but not IoT Core.  To detect that it was IoT Core we would need to change sys.platform from "win32" to "win-iot", or add some other property or heuristic to check.

File: Tools/nuget/pythonARM32.nuspec (hunk 1 above)
Comment (paulmon) [reply Tools/nuget/pythonARM32.nuspec]: removing the nuget changes

**Pull request merged**